src/hubmap_translation/addl_index_transformations/portal/tests/test_transform.py

```diff
@@ -148,7 +148,8 @@ def raise_for_status(self):
 
 expected_output_doc_unknown_assay = expected_output_doc | {'mapped_data_types': [
     'RNAseq [Salmon]'], 'assay_display_name': [
-    'RNAseq [Salmon]'], 'visualization': False, 'vitessce-hints': ['unknown-assay']}
+    'RNAseq [Salmon]'], 'visualization': False, 'vitessce-hints': ['unknown-assay'],
+    'transformation_errors': ['No soft assay information returned.']}
```

Comment:
The soft-assay endpoint returns an empty json for this [example (58ebb89caf1512e9452d1f9e0e1efa8e)](https://ingest-api.dev.hubmapconsortium.org/assaytype/58ebb89caf1512e9452d1f9e0e1efa8e). Maybe fall back to dataset_type from entity? In this case, we'd end up with 'ATACseq'.

Reply:
I'm not sure I understand the question. This PR already defaults to set the `assay_display_ name` to whatever the `dataset_type` is.